docs/photoniq/vwrs/03-configuration-and-setup/02-configure-waiting-rooms.md

```diff
@@ -1,25 +1,29 @@
 ---
 sidebar_position: 20
-title: Configuring Waiting Rooms
+title: Configuring Waiting Room Parameters
 ---
 
-This page explains what can be configured in each waiting room using API endpoint properties. For more information about how to create a waiting room, refer to [Configure a Domain](configure-domain.md).
+This page explains some parameters to configure when using the VWR using API endpoint properties. These include configurations when:
+
+- Entering the waiting room
+- In the waiting room
+- Exiting the waiting room
 
 ## Entering the Waiting Room
 
 - **Rate Limit:** Set the rate (maximum users per second) at which traffic should be diverted to the waiting room. (`rate_limit` property)
-- **Waiting Room Path:** The URL that the waiting room needs to sit in front of. (`domain_url` property)
+- **Waiting Room Path:** The URL that sits in front of the waiting room (`domain_url` property)
```

Comment:
This property should be called `waitingroom_url`, not `domain_url`. This is my fault, but please change it while at it :) Also, the whole sentence is confusing.

```suggestion
- **Waiting Room URL:** The URL that you are protecting with a waiting room (`waitingroom_url` property)
```